src/routes/auth/change-password/+page.server.ts

```diff
@@ -0,0 +1,37 @@
+import { HTTPStatusBadRequest, HTTPStatusSuccess, type SocialResponse } from "$lib/utils/types";
+import type { Actions } from "@sveltejs/kit";
+
+export const actions = {
+    request_password_reset: async function (event): Promise<SocialResponse<unknown>> {
+        const formData = await event.request.formData();
+        const newPassword = formData.get('newPassword');
+        const confirmNewPassword = formData.get('confirmNewPassword');
+
+        if (!newPassword || !newPassword.toString().trim()) {
```

Comment:
Just convert those passwords to string.
Then compare ===
Don't trim() because password can contain whitespace